Makefile

```diff
@@ -813,8 +851,8 @@ publish_sources: $(COLLECTED_SOURCES)
 	$(QUIET): $@: Succeeded
 
 
-$(LIVE).raw: $(BOOT_PART) $(EFI_PART) $(ROOTFS_IMG) $(CONFIG_IMG) $(PERSIST_IMG) $(BSP_IMX_PART) $(BIOS_IMG) | $(INSTALLER)
-	./tools/prepare-platform.sh "$(PLATFORM)" "$(BUILD_DIR)" "$(INSTALLER)" || :
```

Comment:
I'm curious, why was this `|| :` there?

Reply:
@OhmSpectator I do not really know @rene 's intention here. he allowed this script to fail